docs/conf.py

```diff
@@ -9,6 +9,10 @@
 
 from rocm_docs import ROCmDocs
 
+import os.path
+import sys
+parentdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
```

Comment:
what is the goal of this code?

Reply:
This snippet adds the parent directory to the PYTHON_PATH, there may be a more succinct/better way to achieve this. I will investigate options.